src/core/obj.js

```diff
@@ -2072,14 +2071,14 @@ let ObjectLoader = (function() {
   }
 
   ObjectLoader.prototype = {
-    load() {
-      this.capability = createPromiseCapability();
-      // Don't walk the graph if all the data is already loaded.
-      if (!(this.xref.stream instanceof ChunkedStream) ||
-          this.xref.stream.getMissingChunks().length === 0) {
-        this.capability.resolve();
-        return this.capability.promise;
+   async load() {
```

Comment:
Missing one space of indentation here.

Reply:
Whoops, looks like just pressing <tab> in the editor didn't work there :-)